app/views/tag_wranglings/_wrangler_dashboard.html.erb

```diff
@@ -34,6 +33,7 @@
         <li><%= span_if_current ts("Characters by fandom (%{count})", :count=> @counts[:characters]), tag_wranglings_path(:show => "characters") %></li>
         <li><%= span_if_current ts("Relationships by fandom (%{count})", :count=> @counts[:relationships]), tag_wranglings_path(:show => "relationships") %></li>
         <li><%= span_if_current ts("Freeforms by fandom (%{count})", :count=> @counts[:freeforms]), tag_wranglings_path(:show => "freeforms") %></li>
+        <li><%= span_if_current(ts("Unsorted Tags (%{count})", :count => UnsortedTag.count), unsorted_tags_path) %></li>
```

Comment:
Can you change this to the newer `count: UnsortedTag.count` syntax? Then I think we can slip this one right on in to 0.9.63.